src/uct/base/uct_iface.h

```diff
@@ -530,5 +530,24 @@ size_t uct_iov_total_length(const uct_iov_t *iov, size_t iovcnt)
     return total_length;
 }
 
+/**
+ * Used to call uct_iov_total_length() in a non-release configuration
+ */
+#if (ENABLE_STATS || ENABLE_PARAMS_CHECK)
+
+#define UCT_IOV_TOTAL_LENGTH_CHECK_IMPL(_total_length, _iov, _iovcnt) \
+    _total_length = uct_iov_total_length(_iov, _iovcnt)
+
+#define UCT_IOV_TOTAL_LENGTH_CHECK(_total_length, _iov, _iovcnt) \
+    size_t _total_length; \
+    UCT_IOV_TOTAL_LENGTH_CHECK_IMPL(_total_length, _iov, _iovcnt)
```

Comment:
@yosefe From your point of view, what is better: having two macroses or implement one macro with first parameter as type like
```
#define UCT_IOV_TOTAL_LENGTH_CHECK_IMPL(_lenght_type, _total_length, _iov, _iovcnt) \
_lenght_type _total_length = uct_iov_total_length(_iov, _iovcnt)

```
and use it, for some code where `lenght` defined somewhere like: 
`UCT_IOV_TOTAL_LENGTH_CHECK_IMPL(, length, iov, iovcnt);
`